gc/gc-repository-jdbc/src/test/java/org/projectnessie/gc/contents/jdbc/TestJdbcPersistenceUnit.java

```diff
@@ -24,6 +24,7 @@
 
 public class TestJdbcPersistenceUnit {
   @Test
+  @SuppressWarnings("InlineMeInliner")
```

Comment:
if this code is considered java11 then we should use the new api here instead probably

Reply:
Yea, but it's just a test